controls/charts/ReadMe.md

```diff
@@ -17,26 +17,22 @@ npm install @syncfusion/ej2-charts
 ## Components included
 
 Following list of components are available in the package
-*	Chart  - Feature-rich chart control with built-in support for over 25 chart types, technical indictors, trendline, zooming, tooltip, selection, crosshair and trackball. 
```

Comment:
remove the extra space and add the platform name before the control name

Reply:
modified